ampd/src/handlers/solana_verify_msg.rs

```diff
@@ -152,19 +160,25 @@ where
         }
 
         println!("222222222222222222222222222");
-        // if !participants.contains(&self.worker) {
-        //     println!("DONT CONTAIN");
-        //     return Ok(());
-        // }
+        if !participants.contains(&self.worker) {
+            println!("DONT CONTAIN");
+            return Ok(());
+        }
 
-        // TODO: Uncomment when using fake workers
-        if !participants.contains(&String::from(
-            "axelar17lqysp4lka9h6cyw8enxhw20t9f855zfw3k3xg",
-        )) {
-            println!("KOR222222222222222222222222222");
+        let latest_block_height = *self.latest_block_height.borrow();
+        if latest_block_height >= expires_at {
+            info!(poll_id = poll_id.to_string(), "skipping expired poll");
```

Comment:
I added this latest block height check, because on my mind this is axelar related and the rest of the handlers have it. Correct me please if am wrong.

Reply:
Makes sense.